app/views/property_owners/shared/_links.html.erb

```diff
@@ -1,21 +1,26 @@
 <%- if controller_name != 'sessions' %>
-  <%= link_to "Log in", new_session_path(resource_name) %><br />
+  <%= link_to I18n.t('devise.sessions.new.sign_in_property_owner'), new_session_path(resource_name) %><br />
 <% end %>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+  <div class="text-center mb-1">
+    <%= link_to new_registration_path(resource_name), class: "text-decoration-none btn hover-strong" do %>
+      <%= image_tag "add-admin", size: "18" %>
+      <span class="text-uppercase fs-7"><%= I18n.t('devise.registrations.new.sign_up') %></span>
+    <% end %>
+  </div>
 <% end %>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
+  <%#= link_to I18n.t('devise.shared.links.forgot_your_password'), new_password_path(resource_name) %>
```

Comment:
Era para essa linha estar comentada? 🤔 
Se não for para exibir nada, por que não apagar a linha? 🤔

Reply:
Visto que o 'forgot your password'  não vai funcionar sem o mailer e como esse código é um padrão do devise, optamos por deixar comentado, assim como ocorre nas suas controllers. Também pensei que a funcionalidade possa ser necessária no futuro. Mas admito que não sei o que é mais correto a se fazer. 🤔 Qual a opinião de vocês?

Reply:
Acredito que a alteração proposta pelo Nathanael só afeta o label do link e não alteraria a funcionalidade de forma geral...